src/ert/job_queue/queue.py

```diff
@@ -441,27 +430,23 @@ def changes_without_transition(self) -> Tuple[Dict[int, str], List[JobStatus]]:
 
     def add_dispatch_information_to_jobs_file(
         self,
-        ens_id: str,
-        dispatch_url: str,
-        cert: Optional[str],
-        token: Optional[str],
         experiment_id: Optional[str] = None,
     ) -> None:
         for q_index, q_node in enumerate(self.job_list):
             cert_path = f"{q_node.run_path}/{CERT_FILE}"
-            if cert is not None:
+            if self._ee_cert is not None:
```

Comment:
Was it not clear when conninfo was passed directly into the function as a parameter?

Reply:
The only thing is that Queue does not need to know about connection setup between ee and job_runner; hence my comment on passing it rather as a function params :shrug:

Reply:
That is correct, I am removing that "feature" to have different connection info.